### ISSUE TITLE
Fix request: false positive on AvoidConstantsInInterface

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#88]
false positive, fields in member classes:
```
public interface Foo {
    @Data
    class UserAccountInformationData { 
        UserAccountInformationAccounts data; 
    }
}
```

**Comment by jborgers** [comment]
fixed
